src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor.cs

```diff
@@ -25,31 +25,40 @@ public partial class StateColumnDisplay
     [Inject]
     public required IStringLocalizer<Columns> Loc { get; init; }
 
+    internal static string? GetResourceStateTooltip(ResourceViewModel resource, IStringLocalizer<Columns> loc)
+    {
+        return GetResourceStateTooltip(
+            resource,
+            loc[Columns.StateColumnResourceExitedUnexpectedly].Value,
+            loc[Columns.StateColumnResourceExited].Value,
+            loc[nameof(Columns.RunningAndUnhealthyResourceStateToolTip)]);
+    }
+
     /// <summary>
     /// Gets the tooltip for a cell in the state column of the resource grid.
     /// </summary>
     /// <remarks>
     /// This is a static method so it can be called at the level of the parent column.
     /// </remarks>
-    public static string? GetResourceStateTooltip(ResourceViewModel resource, IStringLocalizer<Columns> Loc)
+    internal static string? GetResourceStateTooltip(ResourceViewModel resource, string exitedUnexpectedlyTooltip, string exitedTooltip, string runningAndUnhealthyTooltip)
```

Comment:
Did you consider mocking an `IStringLocalizer` rather than breaking this into two methods? Doing that would ensure the correct resources are being used, and would avoid an extra method with a bit of a strange arg list.

Reply:
Done